.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# Require human review for non-metadata changes.
+
+# Require human review for GitHub workflow and repository configuration.
+/.github/ @web-platform-tests/wpt-core-team @web-platform-tests/admins
+# Require human review for Go code changes.
+/go_test/ @web-platform-tests/wpt-core-team @web-platform-tests/admins
+/go_util/ @web-platform-tests/wpt-core-team @web-platform-tests/admins
+# Require human review for top-level Go code changes and repository configuration.
+/* @web-platform-tests/wpt-core-team @web-platform-tests/admins
```

Comment:
Something I learned while reviewing this:

Beforehand, I wasn't sure if this will limit it to top-level only (and not be recursive)

I saw some similar examples:
- https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/f148440ad17dbeee802ecad39fe17f80f090361d/CODEOWNERS#L9
- https://github.com/box/box-ui-elements/blob/a81229420c1a133145c9899efe908a080f59bd9f/CODEOWNERS#L5


LGTM

Reply:
Indeed, there is an [elaborate comment](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#:~:text=The%20%60docs/*%60%20pattern%20will%20match%20files%20like%0A%23%20%60docs/getting%2Dstarted.md%60%20but%20not%20further%20nested%20files%20like%0A%23%20%60docs/build%2Dapp/troubleshooting.md%60.) in the docs that I've read while working on this.